src/SCRIPTS/BF/CMS/common.lua

```diff
@@ -0,0 +1,120 @@
+local _ = {
```

Comment:
What's the reason to name this `_`? I thought convention in lua was to use `_` for unused values

Reply:
This was just a convenient way of bucketing constants into a single reference object, while limiting keystrokes and improving readability.  Open to suggestions.

Reply:
`CONST`?